keyboards/levinson/keymaps/iarumas/neo2-basic-layout.h

```diff
@@ -0,0 +1,131 @@
+#ifndef NEO2_BASIC_LAYOUT_H
```

Comment:
If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

Reply:
Also, if these match the files in your userspace, then you can actually remove these and use the include to add those.